### PR TITLE
ufal/py-resource-policies-duplicated

### PR DESCRIPTION
**Pull request by Paurikova2**
… bitstream group name

| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0.5  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: data_pump/resourcepolicy.py [hunk 1]
```diff
@@ -69,12 +69,16 @@ def import_resource_policies(community_id_dict,
...
-                    imported += 1
+                # if action is DEFAULT_ITEM_READ, we want to add this resource policy only for group
+                # DEFAULT_ITEM_READ, whose is mapping on the position 1
+                if len(group_list) > 1 and actionId == 10:
+                    params['group'] = group_list[1]
+                else:
+                    params['group'] = group_list[0]
+                response = do_api_post(res_policy_url, params, json_p)
+                response = convert_response_to_json(response)
+                response['id']
```
Comment (milanmajchrak) [inline data_pump/resourcepolicy.py:80]: Why this row?

**Comment by milanmajchrak** [comment]
After testing it is still not working properly:
ITEM:
image
Bitstream: 
image

**Comment by milanmajchrak** [comment]
image

**Comment by vidiecan** [comment]
closing, either working or not valid for the main purpose

**Pull request closed without merging**